package/MDAnalysis/lib/NeighborSearch.py

```diff
@@ -0,0 +1,75 @@
+import numpy
```

Comment:
I added a header comment. I'm not very familiar with the rst docs. If I missed defining some rst specific stuff please tell me. I would also appreciate a link to the appropriate sphinx documentation for this.